### ISSUE TITLE
Speech Bubbles

### ISSUE THREAD

**Issue by BookOwl** [Tymewalk/Slither#87]
These are kind of important. :P

**Comment by skistaddy** [comment]
Are we going to use speech bubbles or just text?

**Comment by BookOwl** [comment]
I think that we need to add the actual bubbles.

**Comment by skistaddy** [comment]
So how will you implement the speech bubbles?

**Comment by Tymewalk** [comment]
@bjskistad Draw both ends of the bubble, then add midsegments in-between for wider text?

**Comment by Tymewalk** [comment]
Expanding on that, we'd have a minimum width for speech bubbles, so we'd know how big to make the ends.

If we used SVGs for speech bubbles we could render those and scale them as needed (the minimum width being scaled based on window size).

**Comment by skistaddy** [comment]
True, but drawing both ends of the bubble and then adding the midsegments, as mentioned above, would require some form of pen, correct?

**Comment by Tymewalk** [comment]
@bjskistad No, we would `blit` them to the screen without pen, although that does sound like a good idea.

**Comment by skistaddy** [comment]
@Tymewalk K.
